sdk/inc/azure/iot/az_iot_adu_client.h

```diff
@@ -317,11 +321,16 @@ typedef struct
 typedef struct
 {
   /**
-   * Name of the component that is expected to handle the step.
+   * Name of the update agent handler type that is expected to handle the step.
+   * @remark For more details on handler types, please see here:
+   * https://learn.microsoft.com/en-gb/azure/iot-hub-device-update/device-update-agent-overview#update-handlers
```

Comment:
I am guessing link checker doesn't validate links within doc comments. Don't specify `en-gb` or even `en-us` and let it route automatically.

```suggestion
   * https://learn.microsoft.com/azure/iot-hub-device-update/device-update-agent-overview#update-handlers
```

@sima-zhu, @weshaggard does the link checker verify links in documentation, outside of markdown files?

Reply:
Yup this guy slipped by :)